### PR TITLE
Promoted Recipes page

### LINKED ISSUE #3227: Promote the Recipes page to one level higher
Instead of being under its current _commitment_ section, it should be its own section.

### PR DESCRIPTION
**Pull request by imaustink**
image

### CODE REVIEW

**Comment by chasenlehara** [comment]
This is close but not quite right. The sitemap in [this comment](https://github.com/canjs/canjs/issues/2752#issuecomment-299529110) makes it a little bit more clear that the goal is to show all the recipes in the nav without having to click on Recipes. It should come out to something like this:
screen shot 2017-05-26 at 2 10 43 pm

**Comment by imaustink** [comment]
Updated the PR :)

File: docs/can-guides/Guides.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 @page guides Guides
 @parent canjs 1
 @group guides/getting-started 0 getting started
-@group guides/experiment 1 experiment
-@group guides/commitment 2 commitment
-@group guides/contribute 3 contribute
-@group guides/upgrade 4 upgrade
+@group guides/experiment 2 experiment
+@group guides/commitment 3 commitment
```
Comment (chasenlehara) [inline docs/can-guides/Guides.md]: I think this should be 4. Right now I’m seeing some weird behavior when switching between pages:
recipes nav bug

**Comment by imaustink** [comment]
Fixed 😃 
canjs-side-nav

**Pull request merged**